docs/source/faq.rst

```diff
@@ -0,0 +1,22 @@
+**************************
+Frequently Asked Questions
+**************************
+
+.. contents:: The following is a list of common questions asked about pyOpenMS.
+
+How can I wrap a new method with pyOpenMS?
+###########################################
+
+Add an entry to ``src/pyOpenMS/pxds/CLASS_NAME.pxd`` with the signature of your new method(s).
+
+
+How can I wrap a new class with pyOpenMS?
+#########################################
+
+Create a new file ``src/pyOpenMS/pxds/CLASS_NAME.pxd`` and use the `procedure outlined <https://github.com/OpenMS/OpenMS/wiki/pyOpenMS#--wrapping-classes>`_. 
```

Comment:
instead of wiki, let's link to the page on pyopenms.readthedocs.

Reply:
done